client/components/gravatar/test/index.jsx

```diff
@@ -1,96 +1,103 @@
 /**
  * External dependencies
  */
-import assert from 'assert';
-import ReactDomServer from 'react-dom/server';
+import { expect } from 'chai';
 import React from 'react';
+import { shallow } from 'enzyme';
 
 /**
  * Internal dependencies
  */
 import { Gravatar } from '../';
 
-/**
- * Pass in a react-generated html string to remove react-specific attributes
- * to make it easier to compare to expected html structure
- * @param {string} string react-generated html
- * @return {string} HTML without react attributes
- */
-function stripReactAttributes( string ) {
-	return string.replace( /\sdata\-(reactid|react\-checksum)\=\"[^\"]+\"/g, '' );
-}
-
-describe( 'Gravatar', function() {
-	const bobTester = {
-		avatar_URL: 'https://0.gravatar.com/avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&d=identicon',
+describe( 'Gravatar', () => {
+	const genericUser = {
+		avatar_URL: 'https://0.gravatar.com/' +
+			'avatar/cf55adb1a5146c0a11a808bce7842f7b?s=96&d=mm',
```

Comment:
Great idea! I added `gravatarHash` but didn't bother with `crypto` since that seemed like overkill and detracting. If the reader is curious about where the hash comes from, I added a link to the docs.